apps/medical-illustration/src/app/app-web-component.ts

```diff
@@ -0,0 +1,119 @@
+import { HttpClient } from '@angular/common/http';
+import { ChangeDetectionStrategy, Component, EventEmitter, inject, Input, OnChanges, Output } from '@angular/core';
+import { NodeMapEntry } from '@hra-ui/components/molecules';
+import { BehaviorSubject, Observable, tap } from 'rxjs';
+
+import { CellEntry, IllustrationData, JsonLd } from './models';
+
+/**
+ * Medical illustration web component
+ */
+@Component({
+  selector: 'hra-root-wc',
+  templateUrl: 'app-web-component.html',
+  styleUrls: ['app-web-component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AppWebComponent implements OnChanges {
+  /** Http client */
+  private readonly http = inject(HttpClient);
+
+  /** Ftu illustrations file: can be url or file object */
+  @Input() lookupSrc: string | JsonLd = '';
+
+  /** Illustration info: can be illustration id or illustration data object */
+  @Input() illustrationSrc: string | IllustrationData = '';
+
+  /** Ontology id to highlight */
+  @Input() highlightId = '';
+
+  /** Emits node data when node hovered */
+  @Output() readonly nodeHovered = new EventEmitter<NodeMapEntry>();
+
+  /** Emits node data when node clicked */
+  @Output() readonly nodeClicked = new EventEmitter<NodeMapEntry>();
+
+  /** Illustration svg url behavior subject */
+  readonly url$ = new BehaviorSubject<string>('');
```

Comment:
The way you are using observables is unnecessary and can be replaced by regular properties

Reply:
Replace url$ with a regular property like you did with mapping